ddl/reorg.go

```diff
@@ -291,6 +292,43 @@ func (w *worker) runReorgJob(rh *reorgHandler, reorgInfo *reorgInfo, tblInfo *mo
 	return nil
 }
 
+// tryToUpdateRealReorgRange updates the reorg info range by the existing records.
+func tryToUpdateRealReorgRange(rh *reorgHandler, reorgInfo *reorgInfo, tbl table.Table) {
+	job, pid, elem, dCtx := reorgInfo.Job, reorgInfo.PhysicalTableID, reorgInfo.currElement, reorgInfo.d
+	var start, end kv.Key
+	if reorgInfo.mergingTmpIdx {
+		start, end = tablecodec.GetTableIndexKeyRange(pid, tablecodec.TempIndexPrefix|elem.ID)
+	} else {
+		currentVer, err := getValidCurrentVersion(dCtx.store)
```

Comment:
It seems that the `currentVer` value is changed. Before this PR, the version is got when `region.first` is true. Maybe we can use `job.SnapshotVer`?

Reply:
If we use `job.SnapshotVer`, we cannot get the latest data.

Reply:
Indeed, it went back. 
But if not, does it feel like whether it is "first" doesn't make much of a difference?

Reply:
OK~